metrics/platform-metrics-reporting/README.md

```diff
@@ -0,0 +1,6 @@
+## Sample Data
```

Comment:
You can use this to generate some data so that you have something to look at in the UI.